src/server/common/dbconn.cpp

```diff
@@ -252,18 +276,18 @@ void dbconn::set_oos_flag(const std::string& uuid, int game_id)
 // therefore for queries that can return multiple rows of multiple columns, implement a class to define how the results should be read
 //
 template<typename... Args>
-void dbconn::get_complex_results(rs_base& base, const std::string& sql, Args&&... args)
+void dbconn::get_complex_results(rs_base& base, const std::string& sql, bool async, Args&&... args)
```

Comment:
Only the private ones that got an async flag so far. Public ones probably get async in the name like the test function?

Reply:
Yeah, that's what I was planning on going with.

Reply:
I mentioned putting the connection as first parameter (or really I meant before the query string, missed the base argument) since that makes the most sense to me and putting it between query string and the associated arguments seems awkward.

Reply:
Alright, updated.